src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -339,6 +340,10 @@ public void onRemoteOperationFinish(RemoteOperation operation, RemoteOperationRe
 
             result.getData();
 
+        } else if(operation instanceof CreateFolderOperation){
+            if(result.isException()){
+                showSnackMessage("Hey");
```

Comment:
:D ... what is the purpose of this?

Reply:
Really Sorry forgot to fix this. Actually was trying to see where the snackbar message was generated.

Reply:
@theScrabi I have fixed it now.